lldb/test/API/lang/swift/stdlib/ContiguousArray/TestContiguousArray.py

```diff
@@ -0,0 +1,39 @@
+# TestContiguousArray.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test that contiguous array prints correctly
+"""
+import lldb
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbtest as lldbtest
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import unittest2
+
+
+class TestContiguousArray(lldbtest.TestBase):
+
+    mydir = lldbtest.TestBase.compute_mydir(__file__)
+
+    @swiftTest
+    def test_frame_contiguous_array(self):
+        """Test that contiguous array prints correctly"""
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, 'Set breakpoint here', lldb.SBFileSpec('main.swift'))
+
+        self.expect("frame variable --dynamic-type run-target",
+                    startstr="""(ContiguousArray<a.Class>) array = {
+  _buffer = {
+    _storage = 1 value {
+      [0] = 0x""")
```

Comment:
This looks good, and also shows that we should implement a data formatter for ContiguousArray that hides the internals.